pallets/ddc-staking/src/weights.rs

```diff
@@ -35,153 +35,306 @@ pub trait WeightInfo {
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: DdcStaking Bonded (r:1 w:1)
+	// Proof Skipped: DdcStaking Bonded (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Ledger (r:1 w:1)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Nodes (r:1 w:1)
+	// Proof Skipped: DdcStaking Nodes (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Providers (r:1 w:1)
+	// Proof Skipped: DdcStaking Providers (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcNodes StorageNodes (r:1 w:0)
+	// Proof Skipped: DdcNodes StorageNodes (max_values: None, max_size: None, mode: Measured)
 	// Storage: Balances Locks (r:1 w:1)
+	// Proof: Balances Locks (max_values: None, max_size: Some(1299), added: 3774, mode: MaxEncodedLen)
 	fn bond() -> Weight {
-		Weight::from_parts(39_000_000_u64, 0)
+		Weight::from_parts(753_000_000_u64, 0)
 			.saturating_add(T::DbWeight::get().reads(6_u64))
 			.saturating_add(T::DbWeight::get().writes(5_u64))
 	}
 	// Storage: DdcStaking Ledger (r:1 w:1)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Storages (r:1 w:0)
+	// Proof Skipped: DdcStaking Storages (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Providers (r:1 w:0)
+	// Proof Skipped: DdcStaking Providers (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcNodes StorageNodes (r:1 w:0)
+	// Proof Skipped: DdcNodes StorageNodes (max_values: None, max_size: None, mode: Measured)
 	// Storage: Balances Locks (r:1 w:1)
+	// Proof: Balances Locks (max_values: None, max_size: Some(1299), added: 3774, mode: MaxEncodedLen)
 	// Storage: System Account (r:1 w:1)
+	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
 	fn unbond() -> Weight {
-		Weight::from_parts(37_000_000_u64, 0)
+		Weight::from_parts(686_000_000_u64, 0)
 			.saturating_add(T::DbWeight::get().reads(6_u64))
 			.saturating_add(T::DbWeight::get().writes(3_u64))
 	}
 	// Storage: DdcStaking Ledger (r:1 w:1)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Providers (r:1 w:0)
+	// Proof Skipped: DdcStaking Providers (max_values: None, max_size: None, mode: Measured)
 	// Storage: Balances Locks (r:1 w:1)
+	// Proof: Balances Locks (max_values: None, max_size: Some(1299), added: 3774, mode: MaxEncodedLen)
 	// Storage: System Account (r:1 w:1)
+	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
 	// Storage: DdcStaking LeavingStorages (r:1 w:0)
+	// Proof Skipped: DdcStaking LeavingStorages (max_values: None, max_size: None, mode: Measured)
 	fn withdraw_unbonded() -> Weight {
-		Weight::from_parts(33_000_000_u64, 0)
+		Weight::from_parts(565_000_000_u64, 0)
```

Comment:
The weights were taken from Rakan's PR and was executed on a dedicated benchmarking machine